drivers/adc/ad9081/ad9081.c

```diff
@@ -229,7 +229,7 @@ static const char *const ad9081_jrx_204c_states[] = {
 static int32_t ad9081_jesd_rx_link_status_print(struct ad9081_phy *phy)
 {
 	int32_t ret, l;
-	uint16_t stat;
+	uint16_t stat = 0;
```

Comment:
Couldn't we initialize it to error status?

Reply:
stat is of unsigned type. therefore negative values are not supported.